refresh-data.sh

```diff
@@ -17,8 +17,8 @@ refresh_data(){
 	./create-mysql-db.sh
 	echo 'Importing refresh db'
 	mysql v1 -u root -p < $refresh_dump_name
-    echo 'Setting up Sites'
-    ./cfgov/manage.py runscript setup_sites
+    echo 'Setting up initial data'
+    ./cfgov/manage.py runscript initial_data
 }
```

Comment:
Is there a reason that `setup_sites` has to exist inside `initial_data` ?  Is it possible for both `initial_data` and `refresh-data` to call that?  I see `refresh-data.sh` (might be time to rename it) as the script to load production data, whereas I see `initial_data` as the script to load made-up data - so they're pretty distinct to me.